tests/functional/lib/helpers.js

```diff
@@ -534,21 +550,17 @@ define([
       .end();
   }
 
-  function fillOutForceAuth(context, password) {
-    return context.remote
-      .setFindTimeout(intern.config.pageLoadTimeout)
+  function fillOutForceAuth(password) {
+    return function () {
+      return this.parent
+        .setFindTimeout(intern.config.pageLoadTimeout)
 
-      .findByCssSelector('#fxa-force-auth-header')
-      .end()
+        .then(testElementExists('#fxa-force-auth-header'))
 
-      .findByCssSelector('input[type=password]')
-        .click()
-        .type(password)
-      .end()
+        .then(type('input[type=password]', password))
 
-      .findByCssSelector('button[type="submit"]')
-        .click()
-      .end();
+        .then(click('button[type="submit"]'));
```

Comment:
I know this just comes from the existing state of affairs, but sometimes we add the quotes and sometimes we don't `type="submit"` vs `type=password`. Does that bother us?